xcode/config_generator/render_xcconfigs.rb

```diff
@@ -0,0 +1,100 @@
+require 'json'
+require 'mustache'
+
+# Constants
+configs_folder_name = "TargetConfigurations"
+standard_dev_team = "D4HA43V467"
+enterprise_dev_team = "228J5MMU7S"
+
+# create config files if needed
+File.new("configs_data.json", 'a')
```

Comment:
will it work without this line?

Reply:
python script move it's outstream to this file, without this line we put a spoke in script wheel

Reply:
don't get it. please, show me what will happen